rn/components/ManageTeam.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import { View, Button, TextInput, Alert } from "react-native";
+import styles from "../stylesheet";
+import { Text, ListItem } from "react-native-elements";
+
+import { useAuth } from "../providers/AuthProvider";
+
+export function ManageTeam({}) {
+  const { user } = useAuth();
+  const [newTeamMember, setNewTeamMember] = useState(null);
+  const [teamMemberList, setTeamMemberList] = useState([]);
+
+  // getTeam calls the backend function getMyTeamMembers to retrieve the
+  // team members of the logged in user's project
+  const getTeam = async () => {
+    try {
+      const teamMembers = await user.functions.getMyTeamMembers([]);
+      setTeamMemberList(teamMembers);
+    } catch (err) {
+      Alert.alert("An error occurred while getting team members", err);
+      throw `an error occurred while getting team members: ${err}`;
```

Comment:
Do we really need to throw here (and in add, remove, etc) or is the UI alert (and perhaps console.warn) enough?

Reply:
Fixed - removed the unneeded throw(s) in this file.